sig/handwritten/app/models/announcement.rbs

```diff
@@ -9,5 +9,12 @@ class Announcement < ApplicationRecord
     #       It should be removed if rbs_rails will support it.
     #       refs: https://github.com/pocke/rbs_rails/pull/268
     def published: () -> self
+
+    # NOTE: Added by ActionText
+    def with_rich_text_content_and_embeds: () -> self
```

Comment:
ActionText automatically adds `#with_rich_text_content_and_embeds`
method to the its relation object.  It is generated via
`has_rich_text :content` declaration in the Announcement class.
 
It's difficult to give a type to the method via rbs_collection.
Therefore this adds a type for it manually.